chart/hyrax/values.yaml

```diff
@@ -200,6 +200,16 @@ worker:
   tolerations: []
   affinity: {}
   resources: {}
+  readinessProbe:
+    enabled: false
+    # command:
+    #  - cat
+    #  - /app/samvera/hyrax-webapp/tmp/sidekiq_process_has_started_and_will_begin_processing_jobs
```

Comment:
could add a comment linking to the wiki bit about creating the initializer too maybe?

Reply:
Yeah good call. I'll do that